model/enduser/registry.yaml

```
@@ -0,0 +1,18 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes the end user.
+    attributes:
+      - id: enduser.id
+        type: string
+        brief: "Unique identifier of an authenticated user in the system."
+        examples: [ 'username' ]
+        stability: development
+      - id: enduser.pseudo.id
+        type: string
+        stability: development
+        brief: >
+          Pseudonymous identifier of an end user. This identifier should be a random value that is not directly linked or associated with the end user's actual identity.
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
```

please add a note on PII
```suggestion
        note: |
          > [!Warning]
          > This field contains sensitive (linkable PII) information.
        examples: ['QdH5CAWJgqVT4rOr0qtumf']
```